### ISSUE TITLE
Can't set region in config

### ISSUE THREAD

**Issue by erikcw** [wallix/awless#37]
```bash
✗ awless config set region
config: missing region. Set it with `awless config set region`
✗ awless config set region us-west-1
config: missing region. Set it with `awless config set region`
```

**Comment by fxaguessy** [comment]
`awless` first install did not run well, and the config database don't contain the region.
To fix the problem, you can remove `awless` config folder with: (for masOS/Linux)

    rm -Rf ~/.awless

In order to prevent this happening again, we will separate the initialization of the database (needed to set config variables) with the initialization of the environment (that loads, for example, the region from the database).

**Comment by fxaguessy** [comment]
Fixed by 083542695371e3fca5c897a5593e76de117fb7eb
